css/jquery-glpi.css

```diff
@@ -130,9 +130,6 @@ html .ui-autocomplete {
    border-width: 1px;
    border-style: solid;
    padding: .2em;
-}
-
-.ui-widget-content .error {
```

Comment:
why this has been removed?

Reply:
It is the exact same selector as the previous one.